spec/venue_spec.rb

```diff
@@ -44,12 +44,45 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+
+  describe '#over_capacity' do
+    it 'returns false if less patrons then capacity' do
+
+      venue = Venue.new('Bluebird', 4)
```

Comment:
You can combine these two test by including multiple expectations. So something like this:
```
it "returns true if venue has more patrons vs capacity" do
      venue = Venue.new('Bluebird', 4)
      venue.add_patron('Mike')
      venue.add_patron('Megan')
      venue.add_patron('Bob')

      expect(venue.over_capacity?).to eq(false)

      venue.add_patron('James')
      venue.add_patron('Cat')

      expect(venue.over_capacity?).to eq(true)
end
```